resource/js/util/interceptor/AbortContinueMarkdownListInterceptor.js

```diff
@@ -0,0 +1,193 @@
+import { BasicInterceptor } from 'crowi-pluginkit';
+
+import * as codemirror from 'codemirror';
+
+import markdownTable from 'markdown-table';
+
+/**
+ * The interceptor that reform markdown table
+ */
+export default class AbortContinueMarkdownListInterceptor extends BasicInterceptor {
```

Comment:
I think it's better that AbortContinueMarkdownListInterceptor is more thin and delegates almost processes to MarkdownListHelper.